changelogs/fragments/compat-jinja-2.10-test.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

This should probably be a  minor change rather than a bugfix.
```suggestion
minor_changes:
```

---

sure, but I'm hoping to backport it

---

We most likely will not backport this.